SimG4Core/CustomPhysics/interface/APrimePhysics.h

```diff
@@ -0,0 +1,40 @@
+#ifndef SIMAPPLICATION_APRIMEPHYSICS_H_
+#define SIMAPPLICATION_APRIMEPHYSICS_H 1_
```

Comment:
Possible multiple include here (`SIMAPPLICATION_APRIMEPHYSICS_H_` vs. `SIMAPPLICATION_APRIMEPHYSICS_H`)

Reply:
Good catch, thank you @forthommel !
I submitted a fix in https://github.com/cms-sw/cmssw/pull/38862